full-vnc/Dockerfile

```diff
@@ -9,6 +9,12 @@ ENV WINDOW_MANAGER="openbox"
 
 USER root
 
+# Make JavaFX display windows with visible content
+RUN echo >> /home/gitpod/.bashrc
```

Comment:
Can you make this in one-line to make the docker layer less.

Reply:
Oh, sorry for dropping the ball here. Totally forgot. On it!